x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/EsqlExpressionTranslators.java

```diff
@@ -67,14 +69,22 @@
 import static org.elasticsearch.xpack.esql.type.EsqlDataTypeConverter.versionToString;
 
 public final class EsqlExpressionTranslators {
+
+    public static Object valueOf(Expression e) {
```

Comment:
Should we use `org.elasticsearch.xpack.esql.core.expression.Foldables.valueOf`?

Reply:
Yeah, I'm surprised that we have more than one valuesOf :-), one is enough, will call `Foldables`.